runtime/src/bank_utils.rs

```diff
@@ -59,3 +62,55 @@ pub fn find_and_send_votes(
             });
     }
 }
+
+/// Logs the measurement values
+pub fn submit_loaded_programs_stats(stats: &Stats, slot: Slot) {
```

Comment:
Can this live in `bank::metrics` instead? And can we call it `report_loaded_programs_metrics`? To match the other function signatures in `bank::metrics`.